_data/experiences/labor.yaml

```diff
@@ -0,0 +1,214 @@
+more_info:
+  - type: Online Exhibit
+    text: Online Exhibit – The David Bacon Photography Archive at Stanford
+    url: https://searchworks.stanford.edu/view/13156376
+  - type: Catalog Record
+    text: Catalog Record - David Bacon photography archive, circa 1985-2019
+    url: https://searchworks.stanford.edu/view/13156376
+  - type: Website
+    text: "David Bacon: Stories, Photographs"
+    url: http://dbacon.igc.org/
+items:
+  - key: workers-assemble
+    url: https://purl.stanford.edu/fy093zm7825
+    title: Workers assemble Toyota Corollas at the New United Motor Manufacturing plant.
+    caption:
+    date: 1999
+    creator: David Bacon
+    collection: "Work & Social Justice: The David Bacon Photography Archive at Stanford"
+    note: In 1999, Bacon photographed workers on the assembly line at the New United Motor Manufacturing, Inc. (NUMMI) plant in Fremont, California. Factory.
+    link:
+    image: https://stacks.stanford.edu/image/iiif/fy093zm7825/autoworkers%20061/full/max/0/default.jpg
+    local_image: local-media/Content/Image-Derivatives/Slideshow-1-Screen-11-Labor/fy093zm7825.jpg
+  - key: workers-assemble-2
+    url: https://purl.stanford.edu/gs668fd0148
+    title: Workers assemble Toyota Corollas at the New United Motor Manufacturing plant.
+    caption:
+    date: 1999
+    creator: David Bacon
+    collection: "Work & Social Justice: The David Bacon Photography Archive at Stanford"
+    note: The NUMMI plant was originally a General Motors plant between 1962 and 1982, which closed and then reopened in 1984 as a partnership between GM and Toyota.
+    link:
+    image: https://stacks.stanford.edu/image/iiif/gs668fd0148/autoworkers%20009/full/max/0/default.jpg
+    local_image: local-media/Content/Image-Derivatives/Slideshow-1-Screen-11-Labor/gs668fd0148.jpg
+  - key: workers-assemble-3
+    url: https://purl.stanford.edu/py915bg9181
+    title: Workers assemble Toyota Corollas at the New United Motor Manufacturing plant.
+    caption:
+    date: 1999
+    creator: David Bacon
+    collection: "Work & Social Justice: The David Bacon Photography Archive at Stanford"
+    note: These photographs document work on the assembly line for the Toyota Corolla.
+    link:
+    image: https://stacks.stanford.edu/image/iiif/py915bg9181/autoworkers%20016/full/max/0/default.jpg
+    local_image: local-media/Content/Image-Derivatives/Slideshow-1-Screen-11-Labor/py915bg9181.jpg
+  - key: workers-assemble-4
+    url: https://purl.stanford.edu/sy022jy9750
+    title: Workers assemble Toyota Corollas at the New United Motor Manufacturing plant.
+    caption: 1999
+    date:
```

Comment:
i think caption and date are swapped here. also, you might be able to just leave off the empty captions entirely...i forget, but i think we do this for the bay area video arcades slideshow too

Reply:
I'll push up a commit to fix the date entry. I left empty keys because it looked like that was the pattern from the other slideshows.